m4/pdns_with_lua_records.m4

```diff
@@ -9,7 +9,9 @@ AC_DEFUN([PDNS_WITH_LUA_RECORDS], [
   AC_MSG_RESULT([$enable_lua_records])
 
   AS_IF([test "x$enable_lua_records" != "xno"], [
-    LIBCURL_CHECK_CONFIG("yes", "7.21.3")
+    LIBCURL_CHECK_CONFIG("yes", "7.21.3",[], [
```

Comment:
please use `[ : ]`, the `sh` on some systems deals badly with empty

Reply:
fixed